code/modules/mob/living/carbon/human/human_damage.dm

```diff
@@ -186,13 +186,15 @@
 	..()
 
 /mob/living/carbon/human/proc/Stasis(amount)
-	if((species.flags & NO_SCAN) || isSynthetic())
+	//if((species.flags & NO_SCAN) || isSynthetic()) vorestation removal
+	if(((species.flags & NO_SCAN) && !istype(species, /datum/species/xenochimera)) || isSynthetic()) //vorestation addition

```

Comment:
I can set in_stasis manually, sure. The line below is the real kicker, 'cause all the things that actually check in_stasis are set to return 0 for NO_SCAN species. Almost tempted to just make them not NO_SCAN given the number of things that check it that don't make sense for chimera and oldstyle cloning isn't a thing any more.